addons/mail/models/mail_message.py

```diff
@@ -237,7 +237,7 @@ def unstar_all(self):
         self.env['bus.bus'].sendone((self._cr.dbname, 'res.partner', self.env.user.partner_id.id), notification)
 
     @api.multi
-    def set_message_starred(self, starred):
+    def set_message_starred(self, starred=None):
```

Comment:
can you change method into toggle_message_starred ? no need to pass argument in toggle method. so can you remove argument ?